admin-dev/themes/new-theme/scss/pages/_orders_view.scss

```diff
@@ -1,4 +1,143 @@
+@import "partials/_order-navigation";
+
 #order-view-page {
+  #historyTabContent {
+    .badge {
+      font-size: 0.875rem;
+      line-height: 1.5;
+      padding: 0 5px;
+      font-weight: 600;
+    }
+  }
+
+  .order {
+    &-actions {
+      display: flex;
+      flex-wrap: wrap;
+    }
+  }
+
+  .add-product {
+    &-inputs {
+      margin-top: 24px;
+      min-width: 120px;
+    }
+
+    &-quantity,
+    &-buttons {
+      .form-group {
+        margin-bottom: 0;
+        margin-top: 24px;
+      }
+    }
+  }
+
+  // Todo: Looks like we don't have any custom select without JS to add in a form
+  .select-status {
+    position: relative;
+
+    .select-status-colored {
+      background: white;
+      color: white;
+      border-radius: 4px;
+      border: none;
+      max-width: 280px;
+      box-shadow: 0 0 10px 0 rgba(0, 0, 0, 0.1);
+    }
+
+    &::after {
+      content: "keyboard_arrow_down";
+      position: absolute;
+      font-family: "Material Icons";
+      cursor: pointer;
+      font-size: 1.25rem;
+      color: white;
+      top: 50%;
+      transform: translateY(-50%);
+      right: 10px;
+      pointer-events: none;
+    }
+
+    &.is-bright .select-status-colored,
+    &.is-bright::after {
+      color: $bright-status-text;
+    }
+  }
+
+  #orderProductsPanel {
+    .btn-product-pack-modal {
+      margin-top: 5px;
+      display: block;
+    }
+
+    .info-block {
+      .col-sm:not(:last-child) {
+        .text-muted {
+          margin-bottom: 4px !important;
+        }
+      }
+    }
+  }
+
+  .configure {
+    &-link {
+      display: block;
+      margin-top: 5px;
+    }
+  }
+
+  #product-pack-modal-table {
+    .cell-product-quantity {
+      .badge {
+        font-size: 100%;
+        min-width: 26px;
+        border-radius: 1000px !important;
```

Comment:
1000px isn't huge ?

Reply:
No as I tested a lot of values, it's common to have rounded at right and left properly